<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryApp</class>
 <widget class="QWidget" name="LibraryApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="search_bar">
       <property name="placeholderText">
        <string>Cauta dupa autor / titlu / editura</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search_btn">
       <property name="text">
        <string>Cautare</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="author_rb">
       <property name="text">
        <string>Autor</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">search_rb_btns</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="title_rb">
       <property name="text">
        <string>Titlu</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">search_rb_btns</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="publisher_rb">
       <property name="text">
        <string>Editura</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">search_rb_btns</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTextEdit" name="result">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="json_rb">
       <property name="text">
        <string>JSON</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">file_type_rb_btns</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="html_rb">
       <property name="text">
        <string>HTML</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">file_type_rb_btns</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="text_rb">
       <property name="text">
        <string>Text</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">file_type_rb_btns</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_as_file_btn">
       <property name="text">
        <string>Salvare ca fisier</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="search_rb_btns"/>
  <buttongroup name="file_type_rb_btns"/>
 </buttongroups>
</ui>
